intelmq/bots/parsers/shadowserver/config.py

```diff
@@ -80,6 +81,7 @@ def get_feed(feedname, logger):
         "Open-XDMCP": open_xdmcp,
         "Sandbox-URL": sandbox_url,
         "Sinkhole-HTTP-Drone": sinkhole_http_drone,
+        "Sinkhole6-HTTP-Drone": sinkhole6_http_drone,
```

Comment:
Here we use the actual email subjects. Please rename it to `IPv6-Sinkhole-HTTP-Drone`